documentation/modules/exploit/windows/local/ntusermndragover.md

```diff
@@ -1,8 +1,8 @@
 ## Vulnerable Application
 
-This module exploits a null pointer deference in win32k triggered by the NTUserMNDragOver syscall.
-After receiving the menu window object returned by the window procedure function, the xxxMNFindWindowFromPoint function does not effectively check the validity of its member tagPOPUPMENU, causing the subsequent MNGetpItemFromIndex function to trigger the NULL pointer deference.
-This module has been tested against Windows 7 x86 SP0 and SP1. Offsets within the solution may need to be adjusted to work with other versions of Windows.
+This module exploits a NULL pointer dereference vulnerability in the win32 NtUserMNDragOver() syscall.
```

Comment:
Reviewing this paragraph this doesn't make sense. The vulnerability isn't in `NtUserMNDragOver()` like you are describing here. Rather it is in `MNGetpItemFromIndex()` which is reachable via `NtUserMNDragOver()`, and the fix was to update `xxxMNFindWindowFromPoint()` to better check the `tagPOPUPMENU` object before passing it on to `MNGetpItemFromIndex()`.

Therefore the following sentence might make more sense here:

`This module exploits a NULL pointer dereference vulnerability in MNGetpItemFromIndex(), which is reachable via a NtUserMNDragOver() system call.`

Reply:
Confirmed resolved.